src/assets/fonts.tsx

```diff
@@ -8,4 +8,26 @@ injectGlobal`
         font-weight: normal;
         font-style: normal;
     }
-`
\ No newline at end of file
+`
+
+const primaryFontFamily = `'ITC Avant Garde Gothic W04', 'AvantGardeGothicITCW01D 731075', 'AvantGardeGothicITCW01Dm', 'Helvetica', 'sans-serif'`,
+
+export const primary = {
+    fontFamily: primaryFontFamily,
+    style: `
+        font-family: ${primaryFontFamily};
+        -webkit-font-smoothing: antialiased;
```

Comment:
Here's the module in `styled-components` that does it https://github.com/styled-components/styled-components/blob/c268b4f62192f22e99283d8f5a41803f489c8468/src/utils/autoprefix.js

Which is using [glamor/autoprefix.js](https://github.com/threepointone/glamor/blob/master/src/autoprefix.js)
I'll debug and see what's going on.